### PR TITLE
Foreman Cockpit documentation

### PR DESCRIPTION
**Pull request by dLobatog**
An initial version of the Foreman Cockpit manual for review. Packages
need to be built so it shouldn't be merged until then.

### CODE REVIEW

File: plugins/foreman_cockpit/1.0/index.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+To install Foreman Cockpit, do the following
+
+    # yum install tfm-rubygem-foreman_cockpit
+
+on Fedora and Red Hat systems or
+
+    # apt-get install ruby-foreman-cockpit
+
+on Debian and Ubuntu systems.
+
```
Comment (mmoll) [inline plugins/foreman_cockpit/1.0/index.md:52]: I'd rather write it like:
For RH based systems (BTW, do we ship Feldora packages at the moment?) do
x
for Debian based systems do
y

File: plugins/foreman_cockpit/1.0/index.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+Restarting Foreman might be needed after installing the package.
+
+# 3. Usage
+
+Using Foreman Cockpit is incredibly easy. It requires no configuration after
+installing the package, so just click on a host where Cockpit is enabled, and
+you will see a Cockpit button on the top bar.
+
+![top bar with cockpit](/static/images/plugins/foreman_cockpit/topbar.png)
```
Comment (mmoll) [inline plugins/foreman_cockpit/1.0/index.md:61]: this image is missing

File: plugins/foreman_cockpit/1.0/index.md (hunk 1 above)
Comment (domcleal) [inline plugins/foreman_cockpit/1.0/index.md:52]: > For RH based systems (BTW, do we ship Feldora packages at the moment?) do

We do, but their name would be "rubygem-foreman_cockpit", not "ruby-foreman-cockpit" - identical to EL, without the prefix.

File: plugins/foreman_cockpit/1.0/index.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    # foreman-installer --enable-foreman-plugin-cockpit
+
+Important note: Executing foreman-installer will re-deploy all foreman-related
+configuration files. In case changes have been made, proceed with manual
+installation.
+
+### 2.1.2 Manual installation
+
+If this is the first plugin you're installing, please see the [plugin
+repository section](plugins/#2.2Packageinstallation) to set up the repository
```
Comment (domcleal) [inline plugins/foreman_cockpit/1.0/index.md]: `({{site.baseurl}}plugins/#2.2...)`

File: plugins/foreman_cockpit/1.0/index.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
+---
+layout: plugin
+title: Foreman Cockpit 1.0 manual
+version: 1.0
+---
+
+# 1. {{ page.title }}
+
+[Foreman Cockpit](https://github.com/theforeman/foreman_cockpit) allows you to see a hosts' Cockpit in Foreman.
```
Comment (domcleal) [inline plugins/foreman_cockpit/1.0/index.md]: `host's`

**Comment by dLobatog** [comment]
Thank you all for the comments, fixed.

File: plugins/foreman_cockpit/1.0/index.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+To install Foreman Cockpit, run the following:
+
+* Red Hat based (RHEL, CentOS...)
+
+      # yum install tfm-rubygem-foreman_cockpit
+
+* Fedora
+
+      # yum install tfm-rubygem-foreman_cockpit
```
Comment (domcleal) [inline plugins/foreman_cockpit/1.0/index.md]: no 'tfm-' prefix

File: plugins/foreman_cockpit/1.0/index.md [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+
+* Debian based (Debian, Ubuntu...)
+
+      # apt-get install ruby-foreman-cockpit
+
+on Debian and Ubuntu systems.
```
Comment (domcleal) [inline plugins/foreman_cockpit/1.0/index.md]: Stray line, since you've already got it on 49?

**Comment by dLobatog** [comment]
For the record - the image is not missing anymore :smile:

**Comment by mmoll** [comment]
merged, gracias @dLobatog!

**Pull request closed without merging**